### ISSUE TITLE
scrollable-region-focusable: false positive for disabled textarea

### ISSUE THREAD

**Issue by MichaelKetting** [dequelabs/axe-core#3691]
### Product

axe-core

### Product Version

3.3.4 ... 4.4.2

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
scrollable-region-focusable should not report for textareas (when they are diabled)

#### Actual
I have a disabled textarea element showing a scrollbar because it contains more text than it has vertical space and I get scrollable-region-focusable in my aXe result

#### How to Reproduce
```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
<head>
<title>Example</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
</head>
<body>
  <textarea name="textarea" rows="2" cols="20"  disabled="disabled" aria-label="Test">
      test
      test
      test
  </textarea>
</body>
</html>
```

#### Additional context
Since the browser does not allow focusing the element, it is not within the web-developer's scope to solve this issue for disabled elements.

see also #3692

**Comment by straker** [comment]
Thanks for the issues. I believe you are correct and we should ignore disabled controls for this rule.
